### PR TITLE
change plot subtitles for Ranked abundance

### PR DESCRIPTION
**Pull request by moontrip**
This will resolve https://github.com/VEuPathDB/web-monorepo/issues/265. 

It is obvious what to do, though I could not find where the plot subtitles are displayed 😅

### CODE REVIEW

**Comment by asizemore** [comment]
haha no worries! They are very subtle :) Here's one using your branch!

Screen Shot 2024-02-06 at 6 06 48 AM

File: packages/libs/eda/src/lib/core/components/computations/plugins/abundance.tsx [hunk 1]
```diff
@@ -87,8 +87,8 @@ export const plugin: ComputationPlugin = {
             <>
               <br />
               <span>
-                Ranked abundance: Variables with {config.rankingMethod} = 0
-                removed. Showing up to the top eight variables.
+                Ranked abundance: Overlay variables with {config.rankingMethod}{' '}
```
Comment (asizemore) [inline packages/libs/eda/src/lib/core/components/computations/plugins/abundance.tsx:90]: Is it possible to make the two subtitles more similar by having this part be `...with {config.rankingMethod} = 0 removed...`? Or alternatively can we change the subtitle at lines 61/62 to be `with {config.rankingMethod}{' '}= 0 removed`?

**Review by asizemore — COMMENTED** [review]
One quick question! Otherwise i tested and it looks good!

File: packages/libs/eda/src/lib/core/components/computations/plugins/abundance.tsx (hunk 1 above)
Comment (moontrip) [reply packages/libs/eda/src/lib/core/components/computations/plugins/abundance.tsx:90]: @asizemore I found it interesting. I have tested and realized that actually that part, {' '}, is automatically added by pre-commit tool(s) (probably prettier). So I cannot change or add that part by myself: it is treated as empty commit. 

Thus we just need to leave it as is

Comment (asizemore) [reply packages/libs/eda/src/lib/core/components/computations/plugins/abundance.tsx:90]: wild. Good to know!

**Pull request merged**